lib/node_modules/@stdlib/math/base/special/modff/docs/types/index.d.ts

```diff
@@ -0,0 +1,79 @@
+/*
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+// TypeScript Version: 4.1
+
+/// <reference types="@stdlib/types"/>
+
+import { Collection } from '@stdlib/types/array';
+
+/**
+* Interface describing `modff`.
+*/
+interface Modff {
+	/**
+	* Decomposes a single-precision floating-point number into integral and fractional parts, each having the same type and sign as the input value.
+	*
+	* @param x - input value
+	* @returns output array
+	*
+	* @example
+	* var parts = modff( 3.14 );
+	* // returns [ 3.0, 0.1400001049041748 ]
+	*/
+	( x: number ): Array<number>;
```

Comment:
Is there any reason to not type the return value as a two-element tuple here?

```suggestion
	( x: number ): [ number, number ];
```

cc @kgryte

Reply:
@Planeshifter No. You originally typed as a `Collection` (ref: https://github.com/stdlib-js/stdlib/commit/6921c387952e3035aebe0ec6e6b223758c4b5371). That was carried over in a subsequent refactoring.